supplementary_style_guide/glossary_terms_conventions/general_conventions/x.adoc

```diff
@@ -9,6 +9,43 @@
 
 *See also*: xref:xemacs[XEmacs]
 
+[discrete]
+[[x86_64]]
+==== image:images/yes.png[yes] x86_64 (noun)
```

Comment:
Do we need to surround the noun in backticks as it refers to the object instance of the arch.

Reply:
The confusing part about this x86_64 format is that it doesn't have a format that is exclusive to either objects or general sentences, like how it is for ARM64 and `arm64`

Reply:
Thanks for replying, @kelbrown20 . NP. Feel free to close this comment.